docs/installation.md

```diff
@@ -21,6 +21,16 @@ helm repo add openfeature https://open-feature.github.io/open-feature-operator/
 helm repo update &&
 helm upgrade --install openfeature openfeature/open-feature-operator
 ```
+
+### GitOps
```

Comment:
+1 for the documentation.

It's worth to add an example for argo with referring to its documentation [1]

[1] - https://argo-cd.readthedocs.io/en/stable/user-guide/diffing/#application-level-configuration